plugins/aws/ec2/unusedEni.js

```diff
@@ -0,0 +1,58 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Unused Elastic Network Interfaces',
+    category: 'EC2',
+    description: 'Ensures that unused AWS Elastic Network Interfaces (ENIs) are removed.',
+    more_info: 'Unused AWS ENIs should be removed to follow best practices and to avoid reaching the service limit.',
+    recommended_action: 'Delete the unused AWS Elastic Network Interfaces',
+    link: 'https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html',
+    apis: ['EC2:describeNetworkInterfaces', 'STS:getCallerIdentity'],
```

Comment:
STS:getCallerIdentity is not being used. It can be removed then.

Reply:
It is being used at line 20